voctocore/lib/videomix.py

```diff
@@ -149,7 +177,8 @@ def recalculateMixerStateFullscreen(self):
             pad = self.padState[idx]
 
             pad.reset()
-            pad.alpha = float(idx == self.sourceA)
+            pad.alpha = float(idx == self.sourceA or (idx == self.sourceB and self.has_alpha_method))
```

Comment:
This expression needs some explanation.
As far as I read it, it makes the b-source visible and moves it to the top, when the a-source is in fullscreen but the b-source has transparency. IS that correct?

That is a very special case that is only applicable if alpha-keying is involved. I think I would like it better if there would be a dedicated conditional branch for the alpha-case.  Something like
```python
if self.has_alpha_method:
  # description of some sort
  pad.alpah = …
  pad.zorder = …
else:
 pad.alpha = float(idx == self.sourceA)
```

That would make the special case very clear and it would also help, if in the future we would like to change the alpha-logic but keep the traditional one or vice versa.